### ISSUE TITLE
Filter by author in News view not working.

### ISSUE THREAD

**Issue by andrebtlima** [CityOfBoston/boston.gov-d8#845]
If you try to filter by author, select a department and click "apply", nothing happens. Also there is a department "JAPHOCRIDAJAWIC" which shouldn't be there. 

<img width="1424" alt="Screen Shot 2019-11-12 at 3 37 34 PM" src="https://user-images.githubusercontent.com/57111927/68708763-61904480-0562-11ea-9559-d73e97fd3f66.png">

**Comment by jimafisk** [comment]
For some reason we're getting a weird long random string taxonomy term in our Contact Vocabulary (https://boston.lndo.site/admin/structure/taxonomy/manage/contact/overview):

![weird_contact_term](https://user-images.githubusercontent.com/5913244/69457711-b2aaf000-0d3b-11ea-958e-5bee3552e073.png)

Unless @davidrkupton knows how to quickly fix this at the migration source, it should be easy enough to delete manually after the migration cutover. We should add it to our todo checklist - @reillyzlab are you maintaining a list of manual fixes after the cutover?

I disabled AJAX to make the other filter options work correctly. However when testing I also noticed a display issue where the terms extend beyond the container:

![display_ish](https://user-images.githubusercontent.com/5913244/69457983-509eba80-0d3c-11ea-8490-d4958c5252fe.png)

I'm looking into fixing the template for this ^
